src/test/groovy/com/causecode/fileuploader/FileUploaderServiceSpec.groovy

```diff
@@ -481,6 +487,13 @@ class FileUploaderServiceSpec extends BaseFileUploaderServiceSpecSetup {
         UFile uFileInstance = UFile.build(type: UFileType.CDN_PUBLIC)
         assert uFileInstance.type == UFileType.CDN_PUBLIC
 
+        and: 'Mocked getProviderInstance method'
+        1 * service.utilitiesService.getProviderInstance(_) >> { String provider ->
```

Comment:
If the method gets called only once, there is no point defining a return value for second call.

Reply:
Done.